modules/web/src/index.js

```diff
@@ -1,17 +1,15 @@
-
-import log from 'log';
-import _ from 'lodash';
-import { fetchConfigs } from 'api-client/api-client';
-import Application from './core/app';
-import defaultConfig from './config';
-
 // importing for side effects only
 import 'bootstrap';
 import 'theme_wso2';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'bootstrap/dist/css/bootstrap-theme.min.css';
 import 'font-ballerina/css/font-ballerina.css';
 
+import log from 'log';
+import _ from 'lodash';
+import { fetchConfigs } from 'api-client/api-client';
+import Application from './core/app';
+import defaultConfig from './config';
```

Comment:
@dakshika @kaviththiranga I Changed order of imports so we can override css rules from libraries we add. This is fine ne?